builds/e2e/templates/nested-parent-vm-setup.yaml

```diff
@@ -1,20 +1,24 @@
+parameters:
+  transportType: 'amqp'
```

Comment:
I think you mean 
`  transportType: ''`

Reply:
Was trying to get default param. I don't think it works like this though